### PR TITLE
Link to RSS feed from the main page

### LINKED ISSUE #153: Make RSS feed more visible
Okay, so now we have an RSS feed, but the only way to find it is to read routes file in the repo :) To improve the situation, the following has to be done:
* add `<link>` header to all pages;
* add a button or something somewhere in the corner.

Any other ideas?

### PR DESCRIPTION
**Pull request by Minoru**
Originally I wanted to use the RSS icon as it's more prominent, but after 2 (two!) hours of trying to make it scale properly with the other `<li>`, I gave up. If any CSS wizards want to take over this, I'm happy to oblige.

Close #153

### CODE REVIEW

**Comment by rexim** [comment]
I'd suggest to keep things simple. If it looks fine on page I'm ok with mergin that. Let me check it locally.

**Comment by rexim** [comment]
So, if I understand correctly this change should look like this:

RSS

Won't the user accidentally click on the RSS instead of add a new quote all the time?

I'm actually thinking about a fixed positioned RSS icon button at the right bottom corner that follows the user's view  while they scroll:

Fixed RSS Icon

@ForNeVeR @Minoru What do you guys think?

**Comment by Minoru** [comment]
Well, putting it on the margin doesn't look good to me—the page becomes "unbalanced". Maybe placing it next to pagination links is better? If user got through the first page, presumably they like our quotes enough to consider a subscription.

**Comment by rexim** [comment]
@Minoru sounds good to me.

**Comment by ForNeVeR** [comment]
Personally I'd place it in the footer, near the copyright info. Usually it's the place for all sorts of technical links, and I'd expect to see it there.

RSS link near the pagination links is also okay.

**Comment by ForNeVeR** [comment]
Also, @Minoru, fell free to contact me if you want to scale or position some shit with CSS, I have the necessary experience.

**Comment by Minoru** [comment]
How do you like it now?

@ForNeVeR, if you can get an image to scale to text's height in [this gist](https://gist.github.com/Minoru/e5142521663ad66a5435a2218987fbad), we'll be able to use an icon in the footer. But I'm pretty happy with the page as it is and am not entirely convinced that icon would look better.

**Comment by rexim** [comment]
@Minoru jsfiddle is much more convenient for such thing than gist https://jsfiddle.net/y9tfrstk/

**Comment by ForNeVeR** [comment]
I'm on it.

**Comment by ForNeVeR** [comment]
Well, guys, there's a simple CSS metric that matches the current font size. It's called `em`. `height: 1em` pretty much does what you want, check this jsfiddle: https://jsfiddle.net/y9tfrstk/3/

```css
ul.menu li img {
  height: 1em;
  vertical-align: text-bottom; // added because looks nice
}
```

**Comment by Minoru** [comment]
R-right… Now I feel kinda stupid for only thinking of HTML height attribute and not of CSS :(

Anyway, thanks @ForNeVeR; the PR is updated with your idea.

**Comment by ForNeVeR** [comment]
@Minoru should we still consider it as a work in progress? Or should we start the review & merge process?

**Comment by Minoru** [comment]
Right; thanks. I think it's ready to be reviewed.

**Comment by rexim** [comment]
@ForNeVeR I have some technical difficulties right now. Could you please take this PR?

**Comment by ForNeVeR** [comment]
Okay, I'll try to review that tonight.

**Comment by rexim** [comment]
Ah! I'm on vacation and totally forgot that everyone else still works! :D Sorry. I guess will be able to review that in a couple of hours once I get my problems resolved.

**Comment by ForNeVeR** [comment]
First of all, I've built the site with these patches, and it looks pretty good. All the functionality works.

File: scalajvm/public/images/feedicon.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<stop  offset="1.0" stop-color="#D95B29"/>
+</linearGradient>
+</defs>
+<rect width="256" height="256" rx="55" ry="55" x="0"  y="0"  fill="#CC5D15"/>
+<rect width="246" height="246" rx="50" ry="50" x="5"  y="5"  fill="#F49C52"/>
+<rect width="236" height="236" rx="47" ry="47" x="10" y="10" fill="url(#RSSg)"/>
+<circle cx="68" cy="189" r="24" fill="#FFF"/>
+<path d="M160 213h-34a82 82 0 0 0 -82 -82v-34a116 116 0 0 1 116 116z" fill="#FFF"/>
+<path d="M184 213A140 140 0 0 0 44 73 V 38a175 175 0 0 1 175 175z" fill="#FFF"/>
+</svg>
```
Comment (ForNeVeR) [inline scalajvm/public/images/feedicon.svg]: What's the legal status of this icon? Have you drawn it yourself or have you taken it somewhere? What's the license?

**Comment by ForNeVeR** [comment]
Overall it looks good, but we need the answer to my question.

File: scalajvm/public/images/feedicon.svg (hunk 1 above)
Comment (Minoru) [inline scalajvm/public/images/feedicon.svg]: I've retrieved it [from WikiMedia](https://commons.wikimedia.org/wiki/File:Feed-icon.svg), where it is claimed to be under the terms of MPL/GPL/LGPL tri-license. The SVG version is drawn by Matt Brett, but there's no mention of which license he attaches to the file.

I can't tell outright what should we do to be able to use the file. Can you?

Comment (rexim) [inline scalajvm/public/images/feedicon.svg]: I just realized that LogList is under Apache license. :D Looks like this LICENSE file was a part of a Play framework template or something. I thought it was under MIT, 'cause I like it better than GPL virus. We need to discuss the licensing of LogList in an appropriate place. I'll create an issue for that.

Regarding the icon, if GPL is not for us, I can draw us a cool RSS icon and we can publish it under CC BY :)

Comment (hagane) [inline scalajvm/public/images/feedicon.svg]: There should not be any legal issues. As [this file's origin](https://www.feedicons.com) states: 

> The feed icon is freely available for general use in connection with web feeds using open web syndication formats, without the need to enter into a trademark license agreement or similar legal arrangement.

Moreso, "tri-license" means "use whatever one of these you can and/or like".

If you're still concerned, you may add MPL 1.1 blurb in a comment somewhere in that file, though it will increase it's size twofold. Alternatively, a separate document listing all third-party files and their terms of use may be created and distributed with the source. Any solution will count as a " location where a user would be likely to look for such a notice", as MPL 1.1 requires.

Comment (Minoru) [inline scalajvm/public/images/feedicon.svg]: >Alternatively, a separate document listing all third-party files and 
>their terms of use may be created and distributed with the source. Any 
>solution will count as a " location where a user would be likely to 
>look for such a notice", as MPL 1.1 requires.

I like the idea, but it's geared towards developers; ordinary user who 
only visits a deployed site won't see these attributions. How about 
adding another page, linked from the footer, where we enumerate all the 
components used together with their licenses?

Comment (ForNeVeR) [inline scalajvm/public/images/feedicon.svg]: I am okay with this option.

Comment (Minoru) [inline scalajvm/public/images/feedicon.svg]: How should we go about implementing it, then? Should I update this PR, 
adding the page, or should this be done as a separate PR? Maybe this 
should be considered a part of #159?

Comment (ForNeVeR) [inline scalajvm/public/images/feedicon.svg]: @Minoru this PR cannot be merged without the third-party license page, so to keep things simple, please include it in this PR.

#159 is a separate issue (although we'll maybe include our own license to the license page later), it has nothing to do with this PR currently.

Comment (rexim) [inline scalajvm/public/images/feedicon.svg]: I'm for this PR.

Comment (Minoru) [inline scalajvm/public/images/feedicon.svg]: Sorry, I forgot to re-assign this back to myself for the duration of the fix.

I've just pushed a commit adding the page you requested. @ForNeVeR, you can proceed with the review now.

File: scalajvm/app/controllers/Licenses.scala [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package controllers
+
+import helpers.ActionWithTx
+import play.api.mvc._
+
+object Licenses extends Controller {
+  def listLicenses() = ActionWithTx {
```
Comment (ForNeVeR) [inline scalajvm/app/controllers/Licenses.scala]: We don't need a transaction to just render a static page. Please use `Action` here instead of `ActionWithTx`.

File: scalajvm/app/views/licenses.scala.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@()
```
Comment (ForNeVeR) [inline scalajvm/app/views/licenses.scala.html]: It seems that this line is not required, let's try to remove that.

File: scalajvm/app/views/licenses.scala.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+@main("Log List", show_menus = false) {
+<h1 class="licenses-header">Licenses of components used in LogList</h1>
```
Comment (ForNeVeR) [inline scalajvm/app/views/licenses.scala.html]: Please add an indentation to this block.

File: scalajvm/app/views/main.scala.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-@(title: String, menu: Html = Html(""), scripts: Html = Html(""))(content: Html)
+@(title: String, menu: Html = Html(""), scripts: Html = Html(""), show_menus: Boolean = true)(content: Html)
```
Comment (ForNeVeR) [inline scalajvm/app/views/main.scala.html]: We use `camelCase` notation for parameters, please change `show_menus` to `showMenus`.

**Comment by ForNeVeR** [comment]
Looks nice, thank you very much. I'll merge and deploy that tonight.
